doc/source/api.rst

```diff
@@ -1,3 +1,5 @@
+.. _api:
+
 .. currentmodule:: nomenclature
 
 API documentation
```

Comment:
I would add also all the other classes as stated in the "Getting started" page "_..(a full list of all classes can be found in API documentation)."_ Maybe also in the same order like in the "Getting started" page.

Reply:
Good point, I was thinking of doing that or leaving it for another PR as some of the doc string are a little rough around the edges and this PR is already long enough. I can just dump them all here though and we can talk about them in a future PR.

Reply:
If it's not a big deal I'd prefer leaving it as is and make adding the rest of the classes and discussing about that into its own PR.